code/src/components/Summary.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+// const reloadSurvey = () => window.location.reload() 
+
+const Summary = (props) => {
+    const { age, actor, username, personality, person } = props
+    return (
+        <section className='wrap'>
+           <div className='content-wrap summary'>
+                <h1>Thank you {username}!</h1>
+                <div className='summary-wrap'>
+                    <p className='bold-text'>Summary of your answers:</p>
+                    <p>{actor} should be the next president.</p>
+                    {personality === true && <p>You checked yourself and think you are a fearless person.</p>}
+                    {personality === false && <p>You don't see yourself as a fearless person.</p>}
+                    {person === true && <p>You think you are judgemental person.</p>}
+                    {person === false && <p>You are not a judgemental person.</p>}
```

Comment:
I think this could be shortened to only 'personality && ....' or 'person &&....' since they are both booleans?